<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mailDialog</class>
 <widget class="QDialog" name="mailDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* ===== QDialog ===== */
QDialog {
    background-color: #112424; /* 어두운 배경 */
    font-family: 'Segoe UI';
    font-size: 14px;
    color: #aef3c0;
    border: 1px solid #33aa88;
    border-radius: 12px;
}

/* ===== QLabel (텍스트: mymail, email:, pass:) ===== */
QLabel#mailLabel,
QLabel#passLabel,
QLabel#mymailLabel {
    color: #aef3c0;
    font-size: 14pt;
    font-weight: bold;
}

/* ===== QLineEdit (입력창) ===== */
QLineEdit {
    border: none;
    background-color: transparent;
    color: #aef3c0;
    padding: 6px 10px;
}

/* ===== QDialogButtonBox (OK / Cancel 버튼) ===== */
QPushButton {
    background-color: transparent;
    color: #aef3c0;
    border: 0.5px solid #1de9b6;
    border-radius: 6px;
    padding: 6px 12px;
    font-family: 'Segoe UI';
    font-size: 12pt;
}
QPushButton:hover {
    background-color: #1b5247;
}
QPushButton:pressed {
    background-color: #0e2924;
}



QDialogButtonBox QPushButton {
    min-width: 100px;
    min-height: 36px;
    font-size: 12pt;
}

QFrame#emailFrame,
QFrame#passFrame {
    border-bottom: 1px solid #1de9b6;
    border-left: 1px solid #1de9b6;
    border-right: 1px solid #1de9b6;
}
QFrame#emailFrame {
    border-top: 1px solid #1de9b6;
    border-top-left-radius: 6px;
    border-top-right-radius: 6px;
}
QFrame#passFrame {
    border-bottom-left-radius: 6px;
    border-bottom-right-radius: 6px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="2,3,2">
   <item>
    <widget class="QLabel" name="mymailLabel">
     <property name="text">
      <string>mymail@gmail.com</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="formFrame">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>30</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>30</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QFrame" name="emailFrame">
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
         <item>
          <widget class="QLabel" name="mailLabel">
           <property name="text">
            <string>email : </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="emailLineEdit"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="passFrame">
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3">
         <item>
          <widget class="QLabel" name="passLabel">
           <property name="text">
            <string>pass  : </string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="passwordLineEdit">
           <property name="echoMode">
            <enum>QLineEdit::EchoMode::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LayoutDirection::LeftToRight</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>mailDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>mailDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
